### PR TITLE
Update owners for structured logging library

### PR DESCRIPTION
**Pull request by serathius**
/kind cleanup

```release-note
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/component-base/logs/OWNERS [hunk 1]
```diff
@@ -2,7 +2,10 @@
 
 approvers:
 - sig-instrumentation-approvers
+- serathius
```
Comment (liggitt) [inline staging/src/k8s.io/component-base/logs/OWNERS:5]: why add the individual alongside the group alias?

Comment (serathius) [reply staging/src/k8s.io/component-base/logs/OWNERS:5]: It's good to keep a direct handles for people that are most knowledgeable about directory to make it easier to find best person to review. Having group works more as a next step when main contributors are OOO and to have bus factor > 1. I have seen this pattern in multiple directories and it was even suggested to me to add myself to this directory. 

Still let me know if I have misunderstood something.

**Review by logicalhan — APPROVED** [review]
/lgtm

**Pull request merged**